python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -2437,10 +2528,60 @@ async def move_rotator(
             function? Default True.
         """
 
-        await self.rem.mtrotator.cmd_move.set_start(
-            position=position, timeout=self.long_timeout
+        rotator_position_tolerance = self.rotator_position_tolerance
+        try:
+            rotator_position_tolerance = (
+                await self.rem.mtrotator.evt_configuration.aget(
+                    timeout=self.fast_timeout
+                )
+            ).positionErrorThreshold
+        except asyncio.TimeoutError:
+            self.log.warning(
+                "Could not get rotator position error threshold. "
+                "Using default value of {self.rotator_position_tolerance}."
+            )
+
+        rotator_position = await self.rem.mtrotator.tel_rotation.aget(
+            timeout=self.fast_timeout
         )
 
+        if abs(rotator_position.actualPosition - position) < rotator_position_tolerance:
+            self.log.warning(
+                f"Current rotator position ({rotator_position.actualPosition:.2f}) "
+                f"already within tolerance of desired position ({position}). "
```

Comment:
Can we add the tolerance to this warning? Mostly since that tolerance change change based on the prior lines.